### ISSUE TITLE
Newline Formatting Options for unsafe Block

### ISSUE THREAD

**Issue by HaloFour** [dotnet/roslyn#19359]
**Version Used**: 

Visual Studio 15.1 (26403.7)

**Steps to Reproduce**:

1. Uncheck all new line options for braces
1. Add the following code:

```cs
static class Program {
    static void Main() {
        unsafe {
        }
    }
}
```

**Expected Behavior**:

The `unsafe` block to have the open brace on the same line as the keyword:

```cs
unsafe {
}
```

**Actual Behavior**:

A new line is inserted between the `unsafe` keyword and the opening brace:

```cs
unsafe
{
}
```

There isn't a section specific for `unsafe` blocks, but I would think that the formatting would be covered by the setting for "control blocks".  This particular setting does apply to `checked` and `unchecked` blocks.  I have not exhaustively tested other scenarios.

**Comment by CyrusNajmabadi** [comment]
PRs welcome.

**Comment by sharwell** [comment]
:memo: I would have expected this option to control it (i.e. no new option should be needed for it):

![image](https://cloud.githubusercontent.com/assets/1408396/25833029/348f7c64-3434-11e7-8155-137df8813eed.png)

**Comment by Pilchie** [comment]
@CyrusNajmabadi already fixed this for 15.3 as part of https://github.com/dotnet/roslyn/pull/16842.
